src/command_helpers/ensure_asset_exists.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Metalware
+  module CommandHelpers
+    module EnsureAssetExists
+      def post_setup
+        super
+        error_if_asset_file_does_not_exist(asset_path)
```

Comment:
There is no need to split this into two methods. It would be simple enough to contain it as one.

Reply:
Also this method is passing the `asset_path` however the method actually accepts the `asset_name`. However this should be resolved by combing them into a single method.